dff/config/superset_dashboard/dashboards/DFF_statistics_dashboard_1.yaml

```diff
@@ -0,0 +1,1315 @@
+        while the second one displays the distribution over time.'
+      height: 20
+      width: 12
+    parents:
+    - ROOT_ID
+    - GRID_ID
+    - TABS-Xoi5oUBxZI
+    - TAB-q4CdGUDlq
+    - ROW-K0fThyfqJq
+    type: MARKDOWN
+  MARKDOWN-HncT4Y_WmV:
+    children: []
+    id: MARKDOWN-HncT4Y_WmV
+    meta:
+      code: '## Flow visit ratio monitor
+
+
+        Thus chart aggregates the ratio of visits for each flow over the whole period
+        of service uptime. The aggregation period can be set to shorter time spans
+        using the filters on the left.'
+      height: 17
+      width: 12
+    parents:
+    - ROOT_ID
+    - GRID_ID
+    - TABS-Xoi5oUBxZI
+    - TAB-6zE8noCIsx
+    - ROW-klrE6Idjlp
+    type: MARKDOWN
+  MARKDOWN-LyRuvHp-7m:
+    children: []
+    id: MARKDOWN-LyRuvHp-7m
+    meta:
+      code: '## Weighted transition graph
+
+
+        The two weighted transition graphs emphasize the most frequently visited nodes
+        and transitions, effectively demonstrating the most popular routes inside
+        the dialog graph.
+
+
+        This can reveal characteristic features of user behavior given the current
+        dialog structure
+
+        that call for changes.
+
+
+        <br />
+
+        '
+      height: 24
+      width: 12
+    parents:
+    - ROOT_ID
+    - GRID_ID
+    - TABS-Xoi5oUBxZI
+    - TAB-JCU6rANFP
+    - ROW-FMNDn-yXPC
+    type: MARKDOWN
+  MARKDOWN-NXluIKVHl5:
+    children: []
+    id: MARKDOWN-NXluIKVHl5
+    meta:
+      code: '# Annotations
+
+
+        Annotations are values that your conversational service derives from user
+        requests and stores in the `Context` object. They are produced by `PROCESSING`
+        functions and by some supplemental services in the `Pipeline`.
+
+
+        To make them available in the dashboard, you need to define a custom extractor
+        function for them (see `Extractor functions` tutorial for more information).
+        The output of that function will then be persisted to the `data` column of
+        the logs table, while the name of the function will be available in the `data
+        key` column. That makes it easy to filter the relevant log entries and use
+        them to build a Superset chart.
+
+
+        It''s important to keep in mind that `data` is a JSON column, which is why
+        you need to specify your own `custom sql` expression to get the relevant value.
+
+
+        ```sql
+
+        JSON_VALUE(data, ''$.key.nested_key'')
+
+        ```
+
+
+        Additionally, you may need to cast the extracted string to a numeric datatype
+        like below to use certain aggregation functions.
+
+        ```sql
+
+        AVG(CAST(JSON_VALUE(data, ''$.key'') AS Int16))
+
+        ```
+
+
+        **Note that the charts below will only display meaningful figures if the respective
+        annotation values have been collected. To get those, run the `sample data
+        provider` tutorial.**'
```

Comment:
I think with the addition of such charts we should include somewhere (in these texts or the user guide or both) an information (either a tutorial or a link to such tutorial) about:
1. How to create charts
2. How to manage charts (how to export them to the yaml format to not lose them after docker container is reset)

Reply:
Updated